src/Microsoft.Data/DataFrame.cs

```diff
@@ -136,6 +200,9 @@ private DataFrame Clone(DataFrameColumn mapIndices = null, bool invertMapIndices
             return new DataFrame(newColumns);
         }
 
+        /// <summary>
+        /// Updates <paramref name="column"/>'s name with <paramref name="newName"/>
```

Comment:
I thought this method was moved to the `Columns` collection in your previous PR, but I guess it wasn't.

What do you think about removing this and leaving the `dataFrame.Columns.SetColumnName` method? We currently have 2 public functions that do the same thing - one on DataFrame, and one on ColumnsCollection.

Reply:
Fixed